src/settings/components/utils.ts

```diff
@@ -496,13 +496,32 @@ export function defaultAsFirstOrderIdentities(identities: Array<Identity>): Arra
 	return result;
 }
 
+/**
+ * Return the requested attribute value as an array if present, or an empty array otherwise.
+ *
+ * @param settings
+ * @param key
+ * @returns
+ */
+export function getAttributeIfPresent(settings: AccountSettings, key: string): Array<string> {
```

Comment:
TBH I don't think this util should always return strings. It should return the proper type of the attribute. This can be achieved through generics, and with a proper update of the type AccountSettingsAttrs